targets/TARGET_NORDIC/TARGET_NRF5/TARGET_SDK13/drivers_nrf/qspi/nrf_drv_qspi.c

```diff
@@ -94,11 +94,6 @@ ret_code_t nrf_drv_qspi_init(nrf_drv_qspi_config_t const * p_config,
                              nrf_drv_qspi_handler_t        handler,
                              void *                        p_context)
 {
-    if (m_cb.state != NRF_DRV_STATE_UNINITIALIZED)
```

Comment:
Since our HAL support setting command spec for each call we need to re-init the QSPI registers with the new values for Nordic implementation. Thats why we have to remove this extra check. It doesn't hurt to reprogram, without that we may not be able to do command spec for each read/write call being made from Driver to HAL.

Reply:
we can remove this if this is bug in their implementation (will be fixed) or find a way how to work with what we specify or reconsider our API choices.

Reply:
What shall we do here? If we call `nrf_drv_qspi_uninit` we set the state to unitialized, and do reinit again. That should be better rather than changing their SDK?

Reply:
I see, I would leave that up to you on what pattern we should follow. What we need is to have the peripheral programmed with right command to be used in qspi communication and this currently is done only by calling nrf init.

Reply:
OK lets do it for now init/unit if that works